parse_hyperparameters.py

```diff
@@ -0,0 +1,35 @@
+import optuna
+import json
+import argparse
+
+
+def value_key(a):
+    if a.value is None:
+        return float('-inf')
+    else:
+        return a.value
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--study-name", help="Study name used during hyperparameter optimization", type=str, default=None)
+parser.add_argument("--storage", help="Database storage path used during hyperparameter optimization", type=str, default=None)
```

Comment:
as a nice-to-have feature, would it be possible to load from a pickle file too? (the default in the zoo)

Reply:
I don't understand your request here?

Reply:
If you're asking about letting users choose between saving the parameters via either pickle or json:

1) These are likely hyperparameters that you'd want to look at/distribute/reuse later. Json is dramatically better for that 
2) Supporting both wouldn't solve any problems I can imagine and would increase the complexity of the code
3) There was some technical problem with pickle here that I ran into when I first created this PR 2 months ago that I can no longer recall because it's been so long

Reply:
> I don't understand your request here?

I meant that by default the zoo store the study object in a pickle file instead of DB like mysql and that would be nice to allow to load such object.